src/main/client/src/components/@pages/RiddleCategoryList.tsx

```diff
@@ -70,19 +73,24 @@ export const RiddleCategoryList: React.FC<RiddleListProps> = (props) => {
       <Heading>Riddleök</Heading>
       <VStack spacing={4} mt={5} align="stretch">
         {riddleCategoryList.map((riddleCategory) => (
-          <Box bg={bg} px={6} py={2} borderRadius="md" _hover={{ bgColor: 'brand.500' }}>
-            <Box onClick={() => onRiddleCategoryClick(riddleCategory.nextRiddle)} style={{ textDecoration: 'none' }}>
-              <Flex align="center" justifyContent="space-between">
-                <Text fontWeight="bold" fontSize="xl">
-                  {riddleCategory.title}
+          <Box
+            bg={bg}
+            px={6}
+            py={2}
+            borderRadius="md"
+            _hover={{ bgColor: useColorModeValue('brand.300', 'brand.700'), transform: 'translateX(0.5em)', cursor: 'pointer' }}
```

Comment:
Transition was missing, but I would dismiss the idea since the bucketlist page does not have this feature.